Tweak documentation for adding a parser

grunweg opened the pull request:
Thanks a lot for the extensive documentation, this made it pretty easy to add two new parsers for languages I care about.

I noticed two small things in the documentation; updating it to reflect this. See commit messages for details.

Wilfred commented on manual/src/adding_a_parser.md:
```diff
@@ -93,10 +93,13 @@ language.
+There may also file names or shebangs associated with your language. Add those by changing the `LANG_EXTENSIONS` constant further below in that file. GitHub's linguist definitions is a useful source of common file extensions.
+
+
+There may also file names or shebangs associated with your language.
```
Looks like you've duplicated a sentence here?

grunweg replied:
Whoops, indeed. Fixed.

grunweg commented:
Found two more places where the docs were out of date; updated accordingly. (And fixed up my second edit, as I noticed the grammar was still a bit off.)

Wilfred commented:
Thanks.
